.github/workflows/tests.yml

```diff
@@ -29,6 +29,14 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: 10
+    - uses: actions/cache@v2
+      with:
+        path: |
+          ~/.npm
+          ~/.cache/ms-playwright
```

Comment:
If browsers should be cached, the browsers.json needs to be added as a cache key, but not sure if they should be cached.

Reply:
Even if the browsers are cached if the browsers mismatch then `node install-from-github.js` will clear it out .

Reply:
+1 to what Max said, if you want to cache browsers the cache key should consist of the minimal required set of files which is browsers.json in our case regardless of whether install step invalidates the cache or not. It should be obvious to a person reading the config why it is ok to cache given directory with a particular key.